src/test/java/seedu/address/testutil/TypicalPersons.java

```diff
@@ -63,6 +63,16 @@ public class TypicalPersons {
             .withEmail("johnd@example.com").withPhone("88765432")
             .withTags("TAs", "CCA", "Acquaintances").build();
 
+    public static final Person KING = new PersonBuilder().withName("King Kong")
```

Comment:
Good idea, since there are only 2 usages of these typical persons. Have removed King and Loki typical persons and  edited JsonAdaptedPersonTest to use the following stub instead:

``` java
    private static final Set<Days> VALID_DAYS_AVAILABLE;

    static {
        try {
            VALID_DAYS_AVAILABLE = ParserUtil.parseDays(Arrays.asList("monday", "tuesday"));
        } catch (ParseException e) {
            throw new RuntimeException(e);
        }
    }
```